src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -1,47 +1,180 @@
 package core.basesyntax;
 
 import java.util.List;
+import java.util.Objects;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```
0 is default value of int